src/app/modules/item/pages/item-chapter-view-group-situation/item-chapter-view-group-situation.component.ts

```diff
@@ -159,4 +185,17 @@ export class GroupSituationChapterViewComponent implements OnChanges, OnDestroy
       });
     }
   }
+
+  onAccess(title: string, _targetId: string, _itemId: string): void {
```

Comment:
Not clear what targetId is as an argument.. If it is a group id, it should be in the naming. If you want to keep "target", it can be "targetGroupId".

Reply:
fixed